frontend/components/LoginForm/LoginForm.tsx

```diff
@@ -3,7 +3,11 @@ import Link from 'next/link'
 import { useRouter } from 'next/router'
 import { useForm, ErrorMessage } from 'react-hook-form'
 import { trackLogIn } from '../../events/users'
-import { useLoginUserMutation, CurrentUserDocument } from '../../generated/graphql'
+import {
+  useLoginUserMutation,
+  CurrentUserDocument,
```

Comment:
I don't think this ended up being used anywhere, did it?

Reply:
`CurrentUserDocument`? Yep it's used on line 43 in the following code block:

```ts
const onSubmit = (data: any) => {
    if (!loading && Object.keys(errors).length === 0) {
      loginUser({
        variables: {
          identifier: data.email,
          password: data.password,
        },
        refetchQueries: [{ query: CurrentUserDocument }],
      })
    }
  }
```